packages/dd-trace/test/appsec/index.spec.js

```diff
@@ -1182,6 +1182,7 @@ describe('IP blocking', function () {
         }).then(() => {
           throw new Error('Not expected')
         }).catch((err) => {
+          expect(err.message).to.not.equal('Not expected')
```

Comment:
unrelated change ? what made me you change that ?

Reply:
Yes, it's unrelated. During the development of this PR I ran into an issue where these tests failed with a different error than expected. But since the `err.message` was never checked, it wasn't clear why it failed. By adding this check it's going to be much easier to debug if these tests ever fail in the future. At the time I also had some changes in this file which was actually related to the PR, but those were later dropped. But I kept this in, because it's an actual improvement to the developer experience

Reply:
just out of curiosity what was the problem that made this test throw ?

Reply:
It somehow ended up in the `throw new Error('Not expected')` then-block, but I can't remember why right now. And it took me a while to figure out that this is what was wrong because it just failed with `Uncaught TypeError: Cannot read properties of undefined (reading 'status')`

Reply:
... as in I got an unexpected exception in the test-code itself, not an assertion failure. This extra check means that the test fails properly due an assertion error, instead of the test-code failing